nss-day-cohort-45/nutyshell-kicking-the-seat-1
# Added CSS files, popup boxes, Friends component, and comments to code.

# Description
Added CSS files, and made Nutshell UI Components all the same size. 
Added Friends component. Can add and delete friend, but value is undefined.
Added popups to the following:
- Add a Friend with the Add Friend Button
- Add this user as a friend via clicking highlighted username in Chat Messages
- Add New Article Button

## Type of change

# Testing Instructions
1. Fetch and serve
2. Login
2. Check if all of the changes are functioning and are displayed

# Checklist:
